### PR TITLE
New version: mmozeiko.wcap version 2024.10.18

### LINKED ISSUE #200009: [Update Request]: mmozeiko.wcap
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

mmozeiko.wcap

### Package Version

/

### Please describe the changes you would like to see

Repo: https://github.com/mmozeiko/wcap/
The existing installer link returns 404 and needs to be updated.
I'm not sure what the exact version number is.

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200009

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200054)

### CODE REVIEW

File: manifests/m/mmozeiko/wcap/2024.10.18/mmozeiko.wcap.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - x86
+  InstallerUrl: https://raw.githubusercontent.com/wiki/mmozeiko/wcap/wcap-x64.exe
+  InstallerSha256: 3F4017FB2224D0D2C023AD376A0148D76D90F75B7713AE7F03012930A57452EC
+- Architecture: arm64
+  UnsupportedOSArchitectures:
+  - arm
+  - x86
+  - x64
+  InstallerUrl: https://raw.githubusercontent.com/wiki/mmozeiko/wcap/wcap-arm64.exe
+  InstallerSha256: 3F4017FB2224D0D2C023AD376A0148D76D90F75B7713AE7F03012930A57452EC
```
Comment (DuckDuckStudio) [inline manifests/m/mmozeiko/wcap/2024.10.18/mmozeiko.wcap.installer.yaml:23]:
```suggestion
  InstallerSha256: 59412C73B2CC0EF4D67F6371151F8553EB6AF3803BAD3B2388FAB0948A71004B
```

**Comment by stephengillie** [comment]
HI @Dragon1573,

Is this intentional?

> === manifests\m\mmozeiko\wcap\2024.10.18 ===
Missing Properties value based on version 2022.11.06:
ReleaseNotesUrl

**Comment by DuckDuckStudio** [comment]
> [!TIP]
> The ReleaseNotesUrl for the previous version no longer seems to be available.
> We seem to be able to replace the ReleaseNotesUrl with: https://github.com/mmozeiko/wcap/blob/main/CHANGELOG.md#20241018

File: manifests/m/mmozeiko/wcap/2024.10.18/mmozeiko.wcap.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  InstallerUrl: https://raw.githubusercontent.com/wiki/mmozeiko/wcap/wcap-x64.exe
+  InstallerSha256: 3F4017FB2224D0D2C023AD376A0148D76D90F75B7713AE7F03012930A57452EC
+- Architecture: arm64
+  UnsupportedOSArchitectures:
+  - arm
+  - x86
+  - x64
+  InstallerUrl: https://raw.githubusercontent.com/wiki/mmozeiko/wcap/wcap-arm64.exe
+  InstallerSha256: 59412C73B2CC0EF4D67F6371151F8553EB6AF3803BAD3B2388FAB0948A71004B
+ManifestType: installer
```
Comment (Dragon1573) [inline manifests/m/mmozeiko/wcap/2024.10.18/mmozeiko.wcap.installer.yaml:24]:
```suggestion
ReleaseNotesUrl: https://github.com/mmozeiko/wcap/blob/main/CHANGELOG.md#20241018
ManifestType: installer
```

**Review by DuckDuckStudio — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**